util/predicates/generic_predicates.go

```diff
@@ -85,44 +85,44 @@ func Any(logger logr.Logger, predicates ...predicate.Funcs) predicate.Funcs {
 			log := logger.WithValues("predicateAggregation", "Any")
 			for _, p := range predicates {
 				if p.UpdateFunc(e) {
-					log.V(4).Info("One of the provided predicates returned true, allowing further processing")
+					log.V(6).Info("One of the provided predicates returned true, allowing further processing")
 					return true
 				}
 			}
-			log.V(4).Info("All of the provided predicates returned false, blocking further processing")
+			log.V(6).Info("All of the provided predicates returned false, blocking further processing")
 			return false
 		},
 		CreateFunc: func(e event.CreateEvent) bool {
 			log := logger.WithValues("predicateAggregation", "Any")
 			for _, p := range predicates {
 				if p.CreateFunc(e) {
-					log.V(4).Info("One of the provided predicates returned true, allowing further processing")
+					log.V(6).Info("One of the provided predicates returned true, allowing further processing")
 					return true
 				}
 			}
-			log.V(4).Info("All of the provided predicates returned false, blocking further processing")
+			log.V(6).Info("All of the provided predicates returned false, blocking further processing")
 			return false
 		},
 		DeleteFunc: func(e event.DeleteEvent) bool {
 			log := logger.WithValues("predicateAggregation", "Any")
 			for _, p := range predicates {
 				if p.DeleteFunc(e) {
-					log.V(4).Info("One of the provided predicates returned true, allowing further processing")
+					log.V(6).Info("One of the provided predicates returned true, allowing further processing")
 					return true
 				}
 			}
-			log.V(4).Info("All of the provided predicates returned false, blocking further processing")
+			log.V(6).Info("All of the provided predicates returned false, blocking further processing")
 			return false
 		},
 		GenericFunc: func(e event.GenericEvent) bool {
 			log := logger.WithValues("predicateAggregation", "Any")
 			for _, p := range predicates {
 				if p.GenericFunc(e) {
-					log.V(4).Info("One of the provided predicates returned true, allowing further processing")
+					log.V(6).Info("One of the provided predicates returned true, allowing further processing")
 					return true
 				}
 			}
-			log.V(4).Info("All of the provided predicates returned false, blocking further processing")
+			log.V(6).Info("All of the provided predicates returned false, blocking further processing")
```

Comment:
I might have misunderstood the PR description, but shouldn't those stay at 4?

My tl;dr was every blocking predicate (i.e. logs in combination with return false) should stay at 4.

Or only a subset of those?

Reply:
@sbueringer those are log entry from All and Any, which provides combinations of predicates (the point number 3 in the PR discription); they should be moved at level 6 because they are "echoing" blocking predicates logs from other predicates (they are redundant).

Reply:
Okay perfect, thx for the explanation!